tests/foreman/api/test_provisioning.py

```diff
@@ -187,126 +186,142 @@ def test_verify_bugzilla_1114640(self, target_sat):
 class TestOperatingSystem:
```

Comment:
Can we either move this OS tests into tests/foreman/api/test_operatingsystem.py or OS CLI tests into provisioning to keep this tests consistent?

Reply:
I think moving it into `tests/foreman/api/test_operatingsystem.py` will be good